test/e2e/QuerySettings.test.js

```diff
@@ -59,6 +59,8 @@ describe('Query Settings', () => {
 
     app.client.click('#gene');
 
+    await sleep(1000);
```

Comment:
why is there a sleep 1 second here?

Reply:
@lolski Workbase needs some time (depends on machine's performance) to process simulated click on keyspace name. This is a flaky test I was talking about — it passes 6/10 times I tested it before the fix.